scipy/stats/_morestats.py

```diff
@@ -1969,6 +2074,25 @@ def rootfunc(ab, xj, N):
         logsf = distributions.gumbel_l.logsf(w)
         sig = array([25, 10, 5, 2.5, 1])
         critical = around(_Avals_gumbel / (1.0 + 0.2/sqrt(N)), 3)
+    elif dist == 'weibull_min':
+        message = ("Critical values of the test statistic are given for the "
+                   "asymptotic distribution. These may not be accurate for "
+                   "samples with fewer than 10 observations. Consider using "
+                   "`scipy.stats.monte_carlo_test`.")
+        if N < 10:
+            warnings.warn(message, stacklevel=2)
+        # [7] writes our 'c' as 'm', and they write `c = 1/m`. Use their names.
+        m, loc, scale = distributions.weibull_min.fit(y)
+        m, loc, scale = _weibull_fit_check((m, loc, scale), y)
+        fit_params = m, loc, scale
+        logcdf = stats.weibull_min(*fit_params).logcdf(y)
+        logsf = stats.weibull_min(*fit_params).logsf(y)
+        c = 1 / m  # m and c are as used in [7]
+        sig = array([0.5, 0.75, 0.85, 0.9, 0.95, 0.975, 0.99, 0.995])
+        critical = _get_As_weibull(c)
+        # Goodness-of-fit tests should only be used to provide evidence
```

Comment:
I did not understand this comment. if we adjust the values from [7], wouldn't this need to be explained in the docstring?

Reply:
I'm not sure if this is significant. The values in [7] are only provided to three decimal places, and they recommend interpolating. I don't think we should return the full precision after interpolation because most of the figures are not significant. This just rounds to three decimal places like the tabulated values, and the comment explains why I'm taking care to round _up_.

Reply:
ok, that makes sense